src/components/Modals/Send/views/Address.tsx

```diff
@@ -108,6 +108,7 @@ export const Address = () => {
             {translate('modals.send.sendForm.sendTo')}
           </FormLabel>
           <AddressInput
+            pe={'9.5'}
```

Comment:
9.5 vs. the default 10 in `<AddressInput />` to give this a lil less padding-end, meaning a lil bit more textarea